src/components/Board/Board.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Task from "../TaskBox/Task";
+import './Board.css';
+import useBoardDrop from "../../hooks/useBoardDrop";
+
+const Board = ({ status, setTasks, todo, readyForDev, inProgress, done }) => {
+    const { dropRef, isOver } = useBoardDrop(status, setTasks);
+
+    let tasksToMap = todo;
+    let text = "TO DO";
+
+    if (status === "readyForDev") {
+        text = "READY FOR DEV";
+        tasksToMap = readyForDev;
+    }
+
+    if (status === "inProgress") {
+        text = "IN PROGRESS";
+        tasksToMap = inProgress;
+    }
+
+    if (status === "done") {
+        text = "DONE";
```

Comment:
maybe to move this logic to custom hook?

Reply:
done